src/test/resources/log4j2-test.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration status="WARN">
+  <Appenders>
+    <Console name="Console" target="SYSTEM_OUT">
+      <PatternLayout pattern="%5p [%d] [%C] ({%t} %F[%M]:%L) - %m%n"/>
+    </Console>
+  </Appenders>
+  <Loggers>
+    <Logger name="com.teragrep" level="info">
+      <AppenderRef ref="Console"/>
+    </Logger>
+    <Root level="warn">
```

Comment:
is it possible to use default info but org.apache.spark.internal.Logging to warn?

Reply:
Seems like so, updated